doc/res-sendFile.md

```diff
@@ -0,0 +1,80 @@
+<!---
+ Copyright (c) 2016 StrongLoop, IBM, and Express Contributors
+ License: MIT
+-->
+
+<h3 id='res.sendFile'>res.sendFile(path [, options] [, fn])</h3>
+
+<div class="doc-box doc-info" markdown="1">
+`res.sendFile()` is supported by Express v4.8.0 onwards
```

Comment:
Again, this is due to having markdown inside of a `<div>` tag.  I should be able to fix this by replacing the backticks with a `<code>` tag.